workloads/kube-burner/common.sh

```diff
@@ -14,6 +14,21 @@ fi
 export TOLERATIONS="[{key: role, value: workload, effect: NoSchedule}]"
 export UUID=${UUID:-$(uuidgen)}
 
+export baremetalCheck=$(oc get infrastructure cluster -o json | jq .spec.platformSpec.type)
```

Comment:
@rsevilla87 can you vet these changes given they impact all kube-burner e2e tests

Reply:
@rsevilla87 re-ping

Reply:
lgtm, seems like it checks the test runs in a BM environment to install python 3.8 later. However I think it should be a software requirement and we shouldn't do it on runtime (we could say that oc and kubectl are also requirements.)